schema/postgresql/v12/visibility/versioned/v1.5/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "1.5",
+  "MinCompatibleVersion": "0.1",
+  "Description": "add root workflow info columns and indices",
+  "SchemaUpdateCqlFiles": [
```

Comment:
We should really rename this to `SchemaUpdateFiles`

Reply:
Ugh, yea... But that's actually not that simple. Need to update the tools to read from the right key...

Reply:
Yeah. It's not really worth changing